test/functional/test-custom-element.js

```diff
@@ -1002,3 +1003,111 @@ describe('CustomElement Loading Indicator', () => {
     });
   });
 });
+
+
+describe('CustomElement Overflow Element', () => {
+
+  class TestElement extends BaseElement {
+  }
+  const ElementClass = document.registerElement('amp-test-overflow', {
+    prototype: createAmpElementProto(window, 'amp-test-overflow', TestElement)
+  });
+
+  const resources = resourcesFor(window);
+  let sandbox;
+  let clock;
+  let element;
+  let overflowElement;
+  let savedMutate;
+  let vsync;
+  let vsyncTasks;
+  let resourcesMock;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+    clock = sandbox.useFakeTimers();
+    resourcesMock = sandbox.mock(resources);
+    element = new ElementClass();
+    element.layoutWidth_ = 300;
+    element.layout_ = Layout.FIXED;
+    overflowElement = document.createElement('div');
+    overflowElement.setAttribute('overflow', '');
+    element.appendChild(overflowElement);
+    vsync = vsyncFor(window);
+    savedMutate = vsync.mutate;
+    vsyncTasks = [];
+    vsync.mutate = mutator => {
+      vsyncTasks.push(mutator);
+    };
+  });
+
+  afterEach(() => {
+    vsync.mutate = savedMutate;
+    resourcesMock.verify();
+    resourcesMock.restore();
+    clock.restore();
+    sandbox.restore();
+    sandbox = null;
+  });
+
+  it('should NOT be initialized by default', () => {
+    expect(element.overflowElement_).to.be.undefined;
+  });
+
+  it('should be initialized to null when absent', () => {
+    element.removeChild(overflowElement);
+    expect(element.getOverflowElement()).to.be.null;
+    expect(element.overflowElement_).to.be.null;
+  });
+
+  it('should be initialized correctly when present', () => {
+    expect(element.getOverflowElement()).to.exist;
+    expect(element.overflowElement_).to.equal(overflowElement);
+    expect(overflowElement).to.not.have.class('amp-visible');
+    expect(overflowElement.getAttribute('tabindex')).to.equal('0');
+    expect(overflowElement.getAttribute('role')).to.equal('button');
+  });
+
+  it('should NOT override role and tabindex', () => {
+    overflowElement.setAttribute('tabindex', '1');
+    overflowElement.setAttribute('role', 'list');
+    expect(element.getOverflowElement()).to.equal(overflowElement);
+    expect(overflowElement.getAttribute('tabindex')).to.equal('1');
+    expect(overflowElement.getAttribute('role')).to.equal('list');
+  });
+
+  it('should noop when overflow is missing', () => {
+    element.removeChild(overflowElement);
+    element.overflowCallback(true, 111);
+    element.overflowCallback(false, 111);
```

Comment:
could you point me to the assertion for this it block? couldnt find it in the beforeEach